### ISSUE TITLE
GroupBox header content alignment problem

### ISSUE THREAD

**Issue by webwarrior06** [MahApps/MahApps.Metro#2255]
Hello MahApps community!
With the advent of ContentControlEx for Groupbox's Header Site ([35f98ae](https://github.com/MahApps/MahApps.Metro/commit/35f98ae88ad8cf52f6b926f3445dbaf9b58cdab1)), I cannot align my header content properly. Here is my code to provide custom content for header;
```xaml
<GroupBox  Margin="10">
        <GroupBox.Header>
            <Grid>
                <Grid.ColumnDefinitions>
                    <ColumnDefinition/>
                    <ColumnDefinition/>
                </Grid.ColumnDefinitions>

                <TextBlock VerticalAlignment="Center">GroupBox Header</TextBlock>
                <Button Grid.Column="1" HorizontalAlignment="Right">Some Functionality</Button>
            </Grid>
        </GroupBox.Header>
    </GroupBox>
```
**Output:**
![grp](https://cloud.githubusercontent.com/assets/3088750/11622777/e2390c6a-9cd0-11e5-9860-8f28b99ccaab.png)

It is expected of **ContentControlEx** to stretch its content(here Grid) like this;
![groupbox1](https://cloud.githubusercontent.com/assets/3088750/11622743/99464b1c-9cd0-11e5-9d91-38f1f0583242.png)

Adding `HorizontalContentAlignment="Stretch"` property in **ContentControlEx** that is located in
`Controls.GroupBox.xaml` solves the problem.

**Comment by punker76** [comment]
@webwarrior06 fixed with ec55eb71aeba828eddc63b4418cd4bf158a07d85

**Comment by webwarrior06** [comment]
Thank you @punker76
